buildkite/bazel-bench/bazel_bench.py

```diff
@@ -30,30 +31,26 @@
 import yaml
 
 
-def _platform_path_str(posix_path):
-    """Converts the path to the appropriate format for platform."""
-    if os.name == "nt":
-        return posix_path.replace("/", "\\")
-    return posix_path
-
-
 # TMP has different values, depending on the platform.
 TMP = tempfile.gettempdir()
 PROJECTS = [
     {
         "name": "Bazel",
         "storage_subdir": "bazel",
         "git_repository": "https://github.com/bazelbuild/bazel.git",
-        "bazel_command": "build ...",
-    }
+        "bazel_command": "build //src:bazel",
+        "active": True
```

Comment:
Do we need that? We typically just comment out disabled projects/test with a reference to the bug.

Reply:
Yeah I added the attr precisely to avoid having to comment out code :)